homeassistant/components/mailbox/__init__.py

```diff
@@ -61,6 +62,23 @@ async def async_setup_platform(
             _LOGGER.error("Unknown mailbox platform specified")
             return
 
+        if p_type not in ["asterisk_mbox", "demo"]:
```

Comment:
What about `asterisk_cdr`?

Reply:
I added it to the list here.
We don't need to create a repair issue for `asterisk_cdr` as the integration will be loaded dynamically by `asterisk_mbox` on https://github.com/home-assistant/core/blob/7e1e12e06fe255cb0e743ef1a6aac2567191ff33/homeassistant/components/asterisk_mbox/__init__.py#L136

`asterisk_cdr` has no setup function, and therefore only the `asterisk_mbox` can trigger to load the mailbox of `asterisk_cdr.`